packages/config/src/lib/internal/resolver.spec.ts

```
@@ -0,0 +1,287 @@
+      },
+    };
+
+    process.env["ITEMS"] = '["override1", "override2"]';
+
+    const actual = resolve({ ...params, config, schema: z.object({ items: z.array(z.string()) }) });
+
+    expect(actual).toEqual({
+      items: ["override1", "override2"],
+    });
+
+    delete process.env["ITEMS"];
```

Biome

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)